src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -391,7 +394,7 @@ private static string CreateArgumentList(IEnumerable<string> args)
                     if (stringBuilder.Length != 0)
                         stringBuilder.Append(' ');
 
-                    stringBuilder.Append(QuoteArgument(arg));
+                    stringBuilder.Append(quoteArgs ? QuoteArgument(arg) : arg);
```

Comment:
I think in the case of special characters you will still want to quote the arguments regardless.

Which of these scenarios are you trying to drive:
1. All Arguments are quoted
2. Quote only args that contain special characters
3. No arguments are quoted

I think your change will allow 3 and 2, but not 1. Quote Arguments will quote the args if they are empty or contain one of the characters its looking for (https://github.com/microsoft/MIEngine/blob/4849c9416da13242db687ceaf68b857d4c83e2fd/src/OpenDebugAD7/MILaunchOptions.cs#L405) . 

It might be useful to clarify the scenario you are enabling in the notes section of the PR.

Reply:
Also this is the wrong place. I think this changes the option for the user's executable args and not what you are trying to change.

Reply:
This is the current description in MIEngine:

> If individual arguments contain characters (such as spaces or tabs), should it be quoted? If 'false', the debugger command will no longer be automatically quoted. \nDefault is 'true'.

I was copying what we already do in MIEngine.
https://github.com/microsoft/MIEngine/blob/9743dd79f858a47f0cecc5eccd52bae55624b8de/src/MICore/LaunchOptions.cs#L2115

I wonder if the AnyCode scenario is not doing the parenthesis arguments correctly.

Reply:
This is putting it into XML that will get unpacked in MIEngine.

Reply:
It is only in PipeTransportOptions.
https://github.com/microsoft/MIEngine/blob/29c6569861b2042c43f04d8d78b2566db2e2ef1f/src/MIDebugPackage/OpenFolderSchema.json#L219

Reply:
> I think in the case of special characters you will still want to quote the arguments regardless.

I think the purpose of `"qutoeArgs":false` is to say "please don't mess with my args, I have this". So I think if qutoeArgs is false, we shouldn't do anything.